codegen-core/src/main/kotlin/io/vrap/rmf/codegen/rendring/CoreCodeGenerator.kt

```diff
@@ -68,6 +73,13 @@ class CoreCodeGenerator @Inject constructor(val dataSink: DataSink
             }.map { dataSink.write(it) }
         }
 
+        if (::allResourceGenerators.isInitialized) {
```

Comment:
This runs the resource generator only if it's initialized?

Reply:
yep, meaning if something was injected there.